ui/pages/settings/settings.container.js

```diff
@@ -97,6 +97,9 @@ const mapStateToProps = (state, ownProps) => {
       : '',
   );
 
+  const shouldRenderExperimentalTab =
+    process.env.TRANSACTION_SECURITY_PROVIDER || process.env.NFTS_V1;
```

Comment:
is there a particular reason we are passing `shouldRenderExperimentalTab` from the container rather than adding directly to `settings.component.js`?

Reply:
My first instinct was that it was being used in multiple places throughout the class component, and that it seemed to make sense as opposed to a header var or declared in a couple places. 

I am not opposed to the idea of the alternative at all, so if thought as to be best I can remove it from the container and shift it to the component

Reply:
This is no longer being passed through the container